### PR TITLE
Feature/conf

### PR DESCRIPTION
**Pull request by luroche**
The first commit to CONFIG_WS.

### CODE REVIEW

File: code/CONFIG/CONFIG_WS.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+from pathlib import Path
+
+#######################       THIRD PARTY IMPORTS        #######################
+
+#######################          MODULE IMPORTS          #######################
+
+#######################          PROJECT IMPORTS         #######################
+
+#######################              ENUMS               #######################
+WS_PATH= f"/home/pi004/GIT_luroche/Battery-Cycler-Calibration"
```
Comment (mariuscrsn) [inline code/CONFIG/CONFIG_WS.py]: Naming convention: CONFIG_WS_PATH

File: code/CONFIG/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from .CONFIG_WS import CONFIG_WS_c, DEFAULT_INFO_EPC, WS_PATH, CONF_PORTS_FILE_PATH, DATA_PATH, INFO_FILE_PATH, V_LS_CALIB_FILE_PATH, V_HS_CALIB_FILE_PATH, I_CALIB_FILE_PATH
+
+__all__ = [
```
Comment (mariuscrsn) [inline code/CONFIG/__init__.py:3]: not export global variables

Comment (mariuscrsn) [reply code/CONFIG/__init__.py:3]: Only CONFIG_WS

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/CONFIG/CONFIG_WS.py (hunk 1 above)
Comment (luroche) [reply code/CONFIG/CONFIG_WS.py]: Done.

File: code/CONFIG/__init__.py (hunk 2 above)
Comment (luroche) [reply code/CONFIG/__init__.py:3]: Done.

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**